omnigibson/scene_graphs/graph_builder.py

```diff
@@ -300,6 +299,8 @@ def _draw_graph():
     figwidth = imgwidth / figdpi
 
     # Draw the graph onto the figure.
+    import matplotlib.pyplot as plt
```

Comment:
what's the reason for this? is there a problem with dependency import ordering?

Reply:
It's a heavy dependency that prolongs load times by a lot. There was also another issue that it caused with one of the install methods, thats resolved now but I left it in since we shouldnt be importing it when most of the viz code wont actually run.